BAZProject/Modules/Commons/ErrorPage/Router/ErrorPageRouter.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ErrorPageRouter.swift
+//  BAZProject
+//
+//  Created by 1058889 on 31/01/23.
+//
+
+import UIKit
+
+class ErrorPageRouter: ErrorPageRouterProtocol {
+    var view: ErrorPageViewProtocol?
+    
+    static func createModule(errorType: ErrorType, titleNavBar: String?) -> UIViewController {
```

Comment:
Puedes optar por una función monádica*, colocando un `model` que a su vez contenga el tipo de error y el título que desees ingresar.


* *`https://dev.to/codebondco/clean-code-principles-function-52hb`